.github/dependabot.yml

```diff
@@ -1,3 +1,5 @@
+﻿# Copyright (c) Microsoft. All rights reserved.
```

Comment:
There's a potential that this will break the dependabot agent, but we don't have a way to check it without merging it. If it breaks, we'll back out the change and expand the exceptions.

Reply:
It didn't break dependabot in accessibility-insights-windows, so we should be ok here